_includes/manuals/1.13/4.3.4.2_isc_dhcp.md

```diff
@@ -75,3 +74,8 @@ subnet 10.1.1.0 netmask 255.255.255.0 {
 
 }
 </pre>
+
+At the moment, foreman proxy does not recognize all configuration syntax and
+parsing is limited to one or more subnet blocks with routers,
+domain-name-servers and range options. These must not be enclosed in pool
```

Comment:
Use backticks around routers, d-n-s, range, and pool to indicate they're config directives.